### ISSUE TITLE
1.8.5 failling to bring private interface up using vbox

### ISSUE THREAD

**Issue by rafamanzo** [mitchellh/vagrant#7613]
After upgrading to 1.8.5 and VirtualBox 5.1 the following stared to happen. Thanks for any help!

### Vagrant version
Vagrant 1.8.5

### Host operating system
Arch Linux (Virtualbox 5.1)

### Guest operating system
Arch Linux

### Vagrantfile
https://github.com/rafamanzo/pcsetup/blob/e2ab89d0bb3af524ca805b11aa2a22e8cde78895/Vagrantfile

### Debug output
https://gist.github.com/rafamanzo/aa6178bfffa1094a15f88eb569aa8002

### Expected behavior
It should generate the script for enabling private network with the correct name.

### Actual behavior
The generated script set the interface name to `Wednesday, 20 July 2016, 03:03:16 PM`

### Steps to reproduce
1. `vagrant up`

**Comment by sethvargo** [comment]
Hi there,

It looks like the command `/bin/ip` returned:

```
Wednesday, 20 July 2016, 03:11:59 PM
Linux 4.3.3-3-ARCH x86_64 GNU/Linux
```

That seems odd. What is the output of `ip` on your system?

**Comment by rafamanzo** [comment]
Thanks for the response. Really odd.

`ip` output looks no different than usual on the host (I've removed my interface's MAC from the output):

```shell
.-(~)---------------------------------------------------------(manzo@WALL-A)-
`--> ip
Usage: ip [ OPTIONS ] OBJECT { COMMAND | help }
       ip [ -force ] -batch filename
where  OBJECT := { link | address | addrlabel | route | rule | neigh | ntable |
                   tunnel | tuntap | maddress | mroute | mrule | monitor | xfrm |
                   netns | l2tp | fou | tcp_metrics | token | netconf }
       OPTIONS := { -V[ersion] | -s[tatistics] | -d[etails] | -r[esolve] |
                    -h[uman-readable] | -iec |
                    -f[amily] { inet | inet6 | ipx | dnet | mpls | bridge | link } |
                    -4 | -6 | -I | -D | -B | -0 |
                    -l[oops] { maximum-addr-flush-attempts } | -br[ief] |
                    -o[neline] | -t[imestamp] | -ts[hort] | -b[atch] [filename] |
                    -rc[vbuf] [size] | -n[etns] name | -a[ll] | -c[olor]}
.-(~)---------------------------------------------------------(manzo@WALL-A)-
`--> ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: enp3s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether xx:xx:xx:xx:xx:xx brd ff:ff:ff:ff:ff:ff
    inet 192.168.0.163/24 brd 192.168.0.255 scope global dynamic enp3s0
       valid_lft 3024sec preferred_lft 3024sec
    inet6 fe80::9b35:5149:4e82:ae6c/64 scope link 
       valid_lft forever preferred_lft forever
```

Judging by the kernel version on the output the `ip` issued was on the guest side which output looks fine as well (I've bought it up withou provate networking just to check the ip):

```shell
vagrant@archbox[~]$ ip
Usage: ip [ OPTIONS ] OBJECT { COMMAND | help }
       ip [ -force ] -batch filename
where  OBJECT := { link | addr | addrlabel | route | rule | neigh | ntable |
                   tunnel | tuntap | maddr | mroute | mrule | monitor | xfrm |
                   netns | l2tp | fou | tcp_metrics | token | netconf }
       OPTIONS := { -V[ersion] | -s[tatistics] | -d[etails] | -r[esolve] |
                    -h[uman-readable] | -iec |
                    -f[amily] { inet | inet6 | ipx | dnet | mpls | bridge | link } |
                    -4 | -6 | -I | -D | -B | -0 |
                    -l[oops] { maximum-addr-flush-attempts } |
                    -o[neline] | -t[imestamp] | -ts[hort] | -b[atch] [filename] |
                    -rc[vbuf] [size] | -n[etns] name | -a[ll] | -c[olor]}
vagrant@archbox[~]$ ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default 
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether 08:00:27:7e:37:2a brd ff:ff:ff:ff:ff:ff
    inet 10.0.2.15/24 brd 10.0.2.255 scope global enp0s3
       valid_lft forever preferred_lft forever
    inet6 fe80::19d5:adfa:f8a2:f58d/64 scope link 
       valid_lft forever preferred_lft forever
```

How is that `ip` call made by Vagrant? I've noticed that always I ssh into the box it displays the following message:

```
Thursday, 21 July 2016, 02:34:48 AM
Linux 4.3.3-3-ARCH x86_64 GNU/Linux
```

It's really familiar with the output Vagrant is getting from `ip`. Which looks to be configured at `/etc/issue` with the following `Arch Linux \r (\l)` by the box.

**Comment by mvanbaak** [comment]
I have the same error.

vagrant 1.8.5
virtualbox 5.1
host: osx 10.12
guest: amazon linux

**Comment by sethvargo** [comment]
Hrm.... Looks like we are somehow grabbing MOTD stuff that _should_ be on stderr, but looks like it's on stdout 😦

**Comment by vadviktor** [comment]
vagrant 1.8.5
virtualbox 5.1.2
host: win 10
guest: ubuntu 14.04.4

In the debug log I see only this snippet to be suspicious:

`INFO subprocess: Command not in installer, restoring original environment...`

context:

```
DEBUG subprocess: Waiting for process to exit. Remaining to timeout: 32000
DEBUG subprocess: Exit status: 0
DEBUG virtualbox_5_1: Searching for SSH port: 22
DEBUG virtualbox_5_1: read_forward_ports: uuid=c9e97438-8e3c-4246-9a69-79e1e1fea800 active_only=false
 INFO subprocess: Starting process: ["C:/Program Files/Oracle/VirtualBox/VBoxManage.exe", "showvminfo", "c9e97438-8e3c-4246-9a69-79e1e1fea800", "--machinereadable"]
 INFO subprocess: Command not in installer, restoring original environment...
DEBUG subprocess: Selecting on IO
DEBUG subprocess: stdout: name="vagrant-rst_default_1469529363067_5691"
```

All other lines are just plain reports of the box setup.

**Comment by agusti-t** [comment]
I am having a similar problem and just posted on stackoverflow about it since I wasn't sure it was a bug report. I'll try @chrisroberts workaround asap.

https://stackoverflow.com/questions/38905448/vagrant-vm-doesnt-get-assigned-ip

**Comment by chrisroberts** [comment]
Fixed via #7676. Thanks for the report!
